vyper/context/validation/module.py

```diff
@@ -94,19 +114,19 @@ def visit_ClassDef(self, node):
         except VyperException as exc:
             raise exc.with_annotation(node) from None
 
-    def visit_Import(self, node):
-        _add_import(node, self.interface_codes, self.namespace)
-
-    def visit_ImportFrom(self, node):
-        _add_import(node, self.interface_codes, self.namespace)
-
     def visit_FunctionDef(self, node):
         func = ContractFunctionType.from_FunctionDef(node)
         try:
             self.namespace["self"].add_member(func.name, func)
         except VyperException as exc:
             raise exc.with_annotation(node) from None
 
+    def visit_Import(self, node):
+        _add_import(node, self.interface_codes, self.namespace)
+
+    def visit_ImportFrom(self, node):
+        _add_import(node, self.interface_codes, self.namespace)
+
```

Comment:
Why is there no difference between these two?

Reply:
They're different node types but the logic to process them is near-identical.  One is `import x`, other is `from .. import x`.

Reply:
I could refactor a bit to handle the not-identical bits prior to calling the function, though.

Reply:
Done.